internal/avgql/client.go

```diff
@@ -0,0 +1,17 @@
+package avgql
+
+import (
+	"context"
+
+	"github.com/aviator-co/av/internal/config"
+	"github.com/shurcooL/graphql"
+	"golang.org/x/oauth2"
+)
+
+func NewClient() *graphql.Client {
+	src := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: config.Av.Aviator.APIToken},
+	)
+	httpClient := oauth2.NewClient(context.Background(), src)
+	return graphql.NewClient("https://api.aviator.co/graphql", httpClient)
```

Comment:
Can you make this configurable? This will probably be useful for testing things locally.

```suggestion
    apiURL := os.Getenv("AV_GRAPHQL_URL")
    if apiURL == "" { apiURL = "https://api.aviator.co/graphql" }
	return graphql.NewClient(apiURL, httpClient)
```